exercises/practice/quadratic-roots/quadratic_roots_example.py

```diff
@@ -0,0 +1,15 @@
+from z3 import *
+
+def roots(a: float, b: float, c: float) -> tuple:
+    """Solves for the two roots of the quadratic expression"""
+    x = Real('x')
+    r1 = Const("r1", RealSort())
+    r2 = Const("r2", RealSort())
```

Comment:
Is there a reason you use
`r1 = Const("r1", RealSort())`
instead of
`r1 = Real("r1")`
To my knowledge, Real("r1") is just a shorthand notation for Const("r1", RealSort())